frontend/routes/package/(_islands)/DownloadChart.tsx

```diff
@@ -13,99 +13,96 @@ interface Props {
 
 export type AggregationPeriod = "daily" | "weekly" | "monthly";
 
-export function DownloadChart(props: Props) {
-  const chartDivRef = useRef<HTMLDivElement>(null);
-  const chartRef = useRef<ApexCharts>(null);
-  const [graphRendered, setGraphRendered] = useState(false);
-
-  const getChartOptions = (
-    isDarkMode: boolean,
-    aggregationPeriod: AggregationPeriod = "weekly",
-  ) => ({
-    chart: {
-      type: "area",
-      stacked: true,
-      animations: {
-        enabled: false,
-      },
-      height: "100%",
-      width: "100%",
-      zoom: {
-        allowMouseWheelZoom: false,
-      },
-      background: "transparent",
-      foreColor: isDarkMode ? "#a8b2bd" : "#515d6c", // jsr-gray-300 for dark mode, jsr-gray-600 for light
+const getChartOptions = (
+  isDarkMode: boolean,
+): ApexCharts.ApexOptions => ({
+  chart: {
+    type: "area",
+    stacked: true,
+    animations: {
+      enabled: false,
     },
-    legend: {
-      horizontalAlign: "center",
-      position: "top",
-      showForSingleSeries: true,
-      labels: {
-        colors: isDarkMode ? "#a8b2bd" : "#515d6c", // jsr-gray-300 for dark mode, jsr-gray-600 for light
-      },
+    height: "100%",
+    width: "100%",
+    zoom: {
+      allowMouseWheelZoom: false,
     },
+    background: "transparent",
+    foreColor: isDarkMode ? "#a8b2bd" : "#515d6c", // jsr-gray-300 for dark mode, jsr-gray-600 for light
+  },
+  legend: {
+    horizontalAlign: "center",
+    position: "top",
+    showForSingleSeries: true,
+    labels: {
+      colors: isDarkMode ? "#a8b2bd" : "#515d6c", // jsr-gray-300 for dark mode, jsr-gray-600 for light
+    },
+  },
+  tooltip: {
+    theme: isDarkMode ? "dark" : "light",
+  },
+  dataLabels: {
+    enabled: false,
+  },
+  stroke: {
+    curve: "straight",
+    width: 1.7,
+  },
+  xaxis: {
+    type: "datetime",
     tooltip: {
-      items: {
-        padding: 0,
+      enabled: false,
```

Comment:
removing `items.padding` was a result of seeing that it is not in the ApexOptions type. From what I can see in the chart, removing it had no visual impact either. Lastly, it is not in the docs: https://apexcharts.com/docs/options/tooltip/#items
so it seemed safe to remove along with using the return type